src/features/ITGovernance/Admin/GRBReview/ParticipantsSection/ParticipantsSection.tsx

```diff
@@ -32,13 +41,80 @@ const ParticipantsSection = ({
   grbReviewStartedAt
 }: ParticipantsSectionProps) => {
   const { t } = useTranslation('grbReview');
+
   const history = useHistory();
   const { pathname } = useLocation();
 
   const isITGovAdmin = useContext(ITGovAdminContext);
 
+  const { showMessage } = useMessage();
+
+  const [reviewerToRemove, setReviewerToRemove] =
+    useState<SystemIntakeGRBReviewerFragment | null>(null);
+
+  const [mutate] = useDeleteSystemIntakeGRBReviewerMutation({
+    refetchQueries: [GetSystemIntakeGRBReviewDocument]
+  });
+
+  const removeGRBReviewer = useCallback(
+    (reviewer: SystemIntakeGRBReviewerFragment) => {
+      mutate({ variables: { input: { reviewerID: reviewer.id } } })
+        .then(() =>
+          showMessage(
+            <Trans
+              i18nKey="grbReview:messages.success.remove"
+              values={{ commonName: reviewer.userAccount.commonName }}
+            />,
+            { type: 'success' }
+          )
+        )
+        .catch(() =>
+          showMessage(t('form.messages.error.remove'), { type: 'error' })
+        );
+
+      // Reset `reviewerToRemove` to close modal
+      setReviewerToRemove(null);
+    },
+    [mutate, showMessage, t, setReviewerToRemove]
+  );
```

Comment:
I am not sure why the modal and function is added here. The section itself does not call the modal to show up.  Reviewer removal functional is already in the `ParticipantsTable` component

Reply:
Oh sorry, I had moved around this modal/mutation quite a bit, trying to find the right place for it, forgot to remove it here